docs/content/concepts/assets/software-defined-assets.mdx

```diff
@@ -368,11 +368,9 @@ Jobs that target assets can materialize a fixed selection of assets each time th
 
 ---
 
----
-
 ## Grouping assets
 
-To help keep your assets tidy, you can organize them into groups. Grouping assets by project, concept, and so on simplifies [keeping track of them in Dagit](#viewing-asset-groups-in-dagit).
+To help keep your assets tidy, you can organize them into groups. Grouping assets by project, concept, and so on simplifies [keeping track of them in Dagit](#viewing-asset-groups-in-dagit). Each asset is assigned to a single group.
```

Comment:
Consider making this something like `Each asset can be assigned to a single group.` The current phrasing (to me, anyhow) makes it sound like this is something that's been done without my needing to do anything.

Reply:
An asset always has a group - the group is "default" if it's not explicitly assigned. I updated the wording to reflect this.